### PR TITLE
Bug 1672759 - Add notifications whenever PRs about GV updates are busted

### PR DESCRIPTION
**Pull request by JohanLorenzo**
Follows up yesterday's meeting https://docs.google.com/document/d/1xXLJVRfLrCUCrTYeXgnKaVAYsDdpzr8N95Xe6Ted7cE/edit#

### Pull Request checklist

- [ ] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/android-components/blob/master/docs/changelog.md) or does not need one
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features

### After merge
- **Milestone**: Make sure issues closed by this pull request are added to the [milestone](https://github.com/mozilla-mobile/android-components/milestones) of the version currently in development.
- **Breaking Changes**: If this is a breaking change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

### CODE REVIEW

File: taskcluster/ac_taskgraph/transforms/complete.py [hunk 1]
```diff
@@ -23,3 +23,70 @@ def fill_dependencies(config, tasks):
...
+                **{
+                    "is-geckoview-nightly-bump": is_geckoview_nightly_bump
+                }
+            )
+        yield task
+
+
+_GECKOVIEW_NIGHTLY_BUMP_BRANCH_PATTERN = re.compile(r"GV-nightly-(?P<date>\d{6})-(?P<time>\d{4})")
+
+def _is_geckoview_nightly_bump(config):
```
Comment (MihaiTabara) [inline taskcluster/ac_taskgraph/transforms/complete.py:45]: ++ for all these checks!

**Review by MihaiTabara — APPROVED** [review]
LGTM!

**Comment by MihaiTabara** [comment]
I think the [broken decision](https://firefox-ci-tc.services.mozilla.com/tasks/cIV70Gu7SdWIzcoEnSLuKg/runs/0/logs/https%3A%2F%2Ffirefox-ci-tc.services.mozilla.com%2Fapi%2Fqueue%2Fv1%2Ftask%2FcIV70Gu7SdWIzcoEnSLuKg%2Fruns%2F0%2Fartifacts%2Fpublic%2Flogs%2Flive.log#L3081) task will be green once we import the `re` module in that transform 👍

**Review by pocmo — APPROVED** [review]

**Comment by JohanLorenzo** [comment]
Yeah, sorry. I'm still working on this PR. I wanted to get it fixed this morning, but today is merge day :)

**Pull request merged**